### PR TITLE
Update modules to their latest nf-core version

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_one_sample.config [hunk 1]
```diff
@@ -50,3 +50,9 @@ params {
     vep_filters          = "https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/reference/hgnc.txt"
     vep_cache_version    = 107
 }
+
+process {
+    withName: '.*FILTERVEP.*' {
+        container  = "docker.io/ensemblorg/ensembl-vep:release_107.0"
```
Comment (asp8200) [inline conf/test_one_sample.config:56]: Just curious to know why you don't update VEP to 110 in the test?

File: nextflow.config [hunk 2]
```diff
@@ -196,6 +196,7 @@ profiles {
     test            { includeConfig 'conf/test.config'            }
     test_full       { includeConfig 'conf/test_full.config'       }
     test_one_sample { includeConfig 'conf/test_one_sample.config' }
+    test_sentieon   { includeConfig 'conf/test_sentieon.config'            }
```
Comment (asp8200) [inline nextflow.config:199]:
```suggestion
    test_sentieon   { includeConfig 'conf/test_sentieon.config'   }
```

(Not that it matters to me, but I know that it does to some people ;-)  )

Comment (ramprasadn) [reply nextflow.config]: Fixed 👍🏻

File: conf/test_one_sample.config (hunk 1 above)
Comment (ramprasadn) [reply conf/test_one_sample.config:56]: I was getting an error about lack of space on device from github CI.. Then, I noticed that the image of vep v110 we use on nf-core modules is about 1.1GB is size, and the one I was using locally had a size of 300MB. So I made the switch to see if that solved CI error, and it looks like it did. 😄

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +30,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Make target bed file optional for WGS mode (Issue [#375](https://github.com/nf-core/raredisease/issues/375)) [#395](https://github.com/nf-core/raredisease/pull/395)
 - Added constraints to block the pipeline from running CollectWgsMetrics on WES samples [#396](https://github.com/nf-core/raredisease/pull/396)
```
Comment (asp8200) [inline CHANGELOG.md:31]:
```suggestion
- Added constraints to block the pipeline from running CollectWgsMetrics on WES samples [#396](https://github.com/nf-core/raredisease/pull/396)
- Updated modules from nf-core [#412](https://github.com/nf-core/raredisease/pull/412)
```

**Pull request merged**